src/index.html

```diff
@@ -11,7 +11,136 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <!-- <h1>HTML Form</h1> -->
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form-field">Surname:
+          <input
+          name="Surname"
+          type="text"
+          required
+          >
+        </label>
```

Comment:
Pay attention on indents. Fix all places 
```suggestion
        <label class="form-field">Surname:
          <input
            name="Surname"
            type="text"
            required
          >
        </label>
```